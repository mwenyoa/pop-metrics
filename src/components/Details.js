import React from 'react';
import { useParams } from 'react-router-dom';
import style from './styles/CatDetails.module.css';

const Details = () => {
  const params = useParams();
  return (
    <section className={style.catDetails}>
      <h2 className={style.categoryname}>Census Details</h2>
      <div className={style.detailMsg}>
        <h3>
          Zambian Population from 1961 - 2018
        </h3>
        <p>{params.code}</p>
      </div>
    </section>
  );
};

export default Details;
