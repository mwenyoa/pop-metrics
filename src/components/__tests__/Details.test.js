import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../../redux/configureStore';
import Details from '../Details';

describe('Render metrics details component', () => {
  test('Test Details component  rendering properly', () => {
    const tree = renderer
      .create(
        <Router>
          <Provider store={store}>
            <Details />
          </Provider>
        </Router>,
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
