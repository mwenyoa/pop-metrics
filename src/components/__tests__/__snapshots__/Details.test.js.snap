// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render metrics details component Test Details component  rendering properly 1`] = `
<section
  className="catDetails"
/>
`;
